src/pages/Flow/Details-Tile.vue

```diff
@@ -70,6 +70,28 @@ export default {
           return obj
         }, {})
     },
+    runConfigDisplayFields() {
+      if (!this.flow.run_config || !this.flow.run_config.type) return []
+
+      let fields = ['type']
+      if (this.flow.run_config.type == 'LocalRun') {
+        fields.push('working_dir')
+      } else if (this.flow.run_config.type == 'DockerRun') {
+        fields.push('image')
+      } else if (this.flow.run_config.type == 'KubernetesRun') {
+        fields.push(
```

Comment:
Do we need to hard code these? I wonder if there might be a way we could pull them dynamically instead?  Maybe have the runConfigFields be the keys of the run_config object (dict) but with type, labels and env filtered out?  Otherwise, we'll need to manually update if/when any changes are made in core.  Could be good reasons - just want to check.

Reply:
Since some fields might be large or undesired, I opted for a hardcoded allow-list for each type. The set of fields are unlikely to change rapidly in core, so I don't think hardcoding it in the UI is a big issue, and it'll be straightforward to add new fields later if needed. It also lets us display fields in nicer logical order, rather than random/alphabetical.

Reply:
Sounds good to me!